.github/CODEOWNERS

```diff
@@ -564,12 +564,11 @@
 
 
 # Release team.
-# TODO(#15381): Add @nithusha21 as owner after 2022-05-01
```

Comment:
Can we close the issue #15381 once this PR is merged? If yes, then let's add "fixes #15381" in PR descrition

Reply:
Done.